pkg/signup/service/signup_service_test.go

```diff
@@ -1608,6 +1608,81 @@ func (s *TestSignupServiceSuite) TestGetSignupReadyConditionStatus() {
 	}
 }
 
+func (s *TestSignupServiceSuite) TestGetSignupBannedUserEmail() {
+	// given
+	s.ServiceConfiguration(configuration.Namespace(), true, "", 5)
+
+	us := s.newBannedUserSignup()
+	us.Name = service.EncodeUserIdentifier(us.Spec.IdentityClaims.PreferredUsername)
```

Comment:
this could be done within `s.newBannedUserSignup()`, couldn't it?

Reply:
Actually it doesn't seem useful at all, unless I'm missing something.

Cleaned in https://github.com/codeready-toolchain/registration-service/pull/404/commits/72752b29c173cfe6137f0ac2a6b56c58a02707c6